frontend/src/components/pages/details/components/DetailsPreview/Breadcrumb.tsx

```diff
@@ -1,27 +1,41 @@
 import Link from 'components/Link';
-import { Fragment } from 'react';
+import { useIntl } from 'react-intl';
+import { cn } from 'services/utils/cn';
 
-const Breadcrumb: React.FC<{ breadcrumb: { label: string; link?: string }[] }> = ({
-  breadcrumb,
-}) => {
+interface BreadcrumbProps {
+  breadcrumb: { label: string; link?: string }[];
+  className?: string;
+}
+
+const Breadcrumb: React.FC<BreadcrumbProps> = ({ breadcrumb, className }) => {
+  const intl = useIntl();
   return (
-    <div className="mt-2 mb-4 desktop:mt-8 desktop:mb-16 flex flex-wrap items-center text-Mobile-C2">
-      {breadcrumb.map(({ label, link }, index) => {
-        const separator = index === 0 ? '>>' : '>';
-        return (
-          <Fragment key={index}>
-            <div className="mx-2">{separator}</div>
-            {link !== undefined ? (
-              <Link href={link} className="text-sm hover:underline">
-                {label}
-              </Link>
-            ) : (
-              label
-            )}
-          </Fragment>
-        );
-      })}
-    </div>
+    <nav
+      className={cn(
+        'mt-2 mb-4 desktop:mt-8 desktop:mb-16 flex flex-wrap items-center text-Mobile-C2',
```

Comment:
I'd prefer.
But just a preference.


```suggestion
        'mt-2',
        'mb-4',
        'desktop:mt-8',
        'desktop:mb-16',
        'flex flex-wrap',
        'items-center',
        'text-Mobile-C2',
```

Reply:
Yes, I agree, there's work to be done to make it easier to read. I'm not sure of the right way, but I am thinking about it. :+1: